### PR TITLE
Add test for checking the uuid structure of the component directory

### PR DESCRIPTION
**Pull request by gaugup**
Since the component directories have uuid based names, adding a test to make sure those component directories are indeed uuids.

### CODE REVIEW

File: responsibleai/tests/test_rai_insights.py [hunk 1]
```diff
@@ -264,6 +265,10 @@ def validate_state_directory(path, manager_type, classes=None):
     all_component_paths = os.listdir(path / manager_type)
     for component_path in all_component_paths:
         # TODO: Add code to check if the component_path is GUID
```
Comment (riedgar-ms) [inline responsibleai/tests/test_rai_insights.py:267]: Is this ToDo now done?

File: responsibleai/tests/test_model_analysis.py [hunk 2]
```diff
@@ -266,6 +267,11 @@ def validate_state_directory(path, manager_type):
     assert manager_type in all_dirs
     all_component_paths = os.listdir(path / manager_type)
     for component_path in all_component_paths:
+        uuid_regex = re.compile(
+            '^[a-f0-9]{8}-?[a-f0-9]{4}-?4[a-f0-9]{3}-?[89ab][a-f0-9]{3}-?[a-f0-9]{12}\Z',
+            re.I)
+        uuid_regex.match(component_path)
```
[the remark this answers is not in the document]
Comment (imatiach-msft) [reply responsibleai/tests/test_model_analysis.py:273]: oh, good point, that does need to get checked

[the remark this answers is not in the document]
Comment (gaugup) [reply responsibleai/tests/test_model_analysis.py]: Done. changed logic to initialize UUID witht he component directory name.

File: responsibleai/tests/test_rai_insights.py (hunk 1 above)
Comment (gaugup) [reply responsibleai/tests/test_rai_insights.py]: Removed it.

**Review by riedgar-ms — APPROVED** [review]

**Review by imatiach-msft — APPROVED** [review]

**Pull request merged**